runtime/cudaq/dynamics/evolution.cpp

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2025 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "common/AnalogHamiltonian.h"
+#include "common/EvolveResult.h"
+#include "common/Logger.h"
+#include "cudaq/operators.h"
+#include "cudaq/schedule.h"
+#include <random>
+#include <sstream>
+#include <string>
+
+namespace cudaq {
+namespace __internal__ {
```

Comment:
```suggestion
namespace cudaq::__internal__ {
```
No need to use Bronze Age C++. :)